<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.534, y1:0.042, x2:0.503, y2:0.954, stop:0.0738636 rgba(0, 45, 79, 255), stop:0.954545 rgba(10, 0, 46, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777000</width>
        <height>16777000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.972, y1:0.961, x2:0, y2:0.011, stop:0.0454545 rgba(46, 88, 198, 90), stop:0.920455 rgba(75, 84, 205, 124));
border-radius: 10px ;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777000</width>
        <height>16777000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.972, y1:0.961, x2:0, y2:0.011, stop:0.0454545 rgba(46, 88, 198, 90), stop:0.920455 rgba(75, 84, 205, 124));
border-radius: 10px ;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777000</width>
        <height>16777000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.972, y1:0.961, x2:0, y2:0.011, stop:0.0454545 rgba(46, 88, 198, 90), stop:0.920455 rgba(75, 84, 205, 124));
border-radius: 10px ;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777000</width>
        <height>16777000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.972, y1:0.961, x2:0, y2:0.011, stop:0.0454545 rgba(46, 88, 198, 90), stop:0.920455 rgba(75, 84, 205, 124));
border-radius: 10px ;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="xscore">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px ;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.852, y1:0.893182, x2:0, y2:0, stop:0.0454545 rgba(17, 70, 205, 187), stop:0.920455 rgba(139, 113, 255, 203));</string>
      </property>
      <property name="text">
       <string>X score : </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777000</width>
        <height>16777000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.972, y1:0.961, x2:0, y2:0.011, stop:0.0454545 rgba(46, 88, 198, 90), stop:0.920455 rgba(75, 84, 205, 124));
border-radius: 10px ;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777000</width>
        <height>16777000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.972, y1:0.961, x2:0, y2:0.011, stop:0.0454545 rgba(46, 88, 198, 90), stop:0.920455 rgba(75, 84, 205, 124));
border-radius: 10px ;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777000</width>
        <height>16777000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.972, y1:0.961, x2:0, y2:0.011, stop:0.0454545 rgba(46, 88, 198, 90), stop:0.920455 rgba(75, 84, 205, 124));
border-radius: 10px ;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777000</width>
        <height>16777000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.972, y1:0.961, x2:0, y2:0.011, stop:0.0454545 rgba(46, 88, 198, 90), stop:0.920455 rgba(75, 84, 205, 124));
border-radius: 10px ;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777000</width>
        <height>16777000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.972, y1:0.961, x2:0, y2:0.011, stop:0.0454545 rgba(46, 88, 198, 90), stop:0.920455 rgba(75, 84, 205, 124));
border-radius: 10px ;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="oscore">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px ;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.852, y1:0.893182, x2:0, y2:0, stop:0.0454545 rgba(17, 70, 205, 187), stop:0.920455 rgba(139, 113, 255, 203));</string>
      </property>
      <property name="text">
       <string>O score :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="draw">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px ;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.852, y1:0.893182, x2:0, y2:0, stop:0.0454545 rgba(17, 70, 205, 187), stop:0.920455 rgba(139, 113, 255, 203));</string>
      </property>
      <property name="text">
       <string>Draw : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="about">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px ;
background-color: qlineargradient(spread:pad, x1:0.778, y1:0.269, x2:0.253, y2:0.835, stop:0.0227273 rgba(125, 219, 254, 183), stop:0.511364 rgba(129, 170, 255, 200), stop:0.982955 rgba(166, 133, 255, 189));</string>
      </property>
      <property name="text">
       <string>    About     </string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QPushButton" name="newgame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px ;
background-color: qlineargradient(spread:pad, x1:0.778, y1:0.269, x2:0.253, y2:0.835, stop:0.0113636 rgba(125, 219, 254, 234), stop:0.511364 rgba(129, 170, 255, 209), stop:0.982955 rgba(166, 133, 255, 198));</string>
      </property>
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QRadioButton" name="player2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>YOU vs Player2</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QRadioButton" name="cpu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>YOU  vs  CPU</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
